rules/CodeQuality/Rector/Catch_/ThrowWithPreviousExceptionRector.php

```diff
@@ -105,15 +105,9 @@ public function refactor(Node $node): ?Node
                 return null;
             }
 
-            $isChanged = $this->refactorThrow($node, $caughtThrowableVariable);
-            return $isChanged;
+            return $this->refactorThrow($node, $caughtThrowableVariable);
```

Comment:
this rectify is invalid, as `$isChanged` is by ref and used outside closure, I will look into it.

Reply:
seems resolved now.